pkg/virt-handler/non-root.go

```diff
@@ -123,6 +123,32 @@ func (d *VirtualMachineController) prepareTap(vmi *v1.VirtualMachineInstance, re
 
 }
 
+func (*VirtualMachineController) prepareVFIO(vmi *v1.VirtualMachineInstance, res isolation.IsolationResult) error {
+	vfioPath := filepath.Join(res.MountRoot(), "dev", "vfio")
```

Comment:
interesting.  what changes the permissions of /dev/vfio/vfio ?

Reply:
ah, that's specific for a given vmi

Reply:
It depends on the host configuration and can be specific for a given container/vmi as everyone might be running under a different user.